fastcrypto/src/groups/mod.rs

```diff
@@ -44,3 +44,14 @@ pub trait Scalar:
 {
     fn rand<R: AllowedRng>(rng: &mut R) -> Self;
 }
+
+pub trait Pair {
```

Comment:
Nits:
1. does it make sense to put a group bound on `Self` or `Other`? 
2. the trait is defined with a dual `Self` assignment, i.e one could define the $G_2 \times G_1 \mapsto G_T$. operation using this same trait just as easily. 
This may or may not be something we want to resolve.

Reply:
1. added it to self & other
2. let's leave it general